### PR TITLE
Add multiple user in a row thought a CSV

### PR DESCRIPTION
**Pull request by zagoa**
* can import a CSV one column 'email'
* fix user deletion
* remove some redux functions

### CODE REVIEW

File: src/main/java/org/oasis_eu/portal/model/user/InvitationRequest.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package org.oasis_eu.portal.model.user;
```
Comment (bobeal) [inline src/main/java/org/oasis_eu/portal/model/user/InvitationRequest.java]: It's better to put in in model.authority (the old name for organization) package.

File: src/main/resources/public/js/components/forms/organization-invitation-form.js [hunk 2]
```diff
@@ -39,86 +42,156 @@ class OrganizationInvitationForm extends React.Component {
...
-                <div className="flex-row-mobile-column">
-                    <div className="flex-row wrapper">
-                        <div className="flex-row">
-                            <label className="label">
-                                {this.context.t('organization.form.email')}
-                                <input name="email" type="email" className="field form-control no-auto" required={true}
-                                       onChange={this.handleChange} value={this.state.email}/>
-                            </label>
+                {/*email input*/}
+                <div className={"form-row-block"}>
```
Comment (bobeal) [inline src/main/resources/public/js/components/forms/organization-invitation-form.js]: why are you wrapping the className values inside {} ?

File: src/main/resources/public/js/components/forms/organization-invitation-form.js [hunk 3]
```diff
@@ -39,86 +42,156 @@ class OrganizationInvitationForm extends React.Component {
...
+        }
     }
 
     render() {
+        const {csvLoading, isFetchingUsers, emailsFromCSV, isLoading, email} = this.state;
+        const submitButton =
+            <button type="submit" className="btn btn-submit" disabled={(isFetchingUsers || csvLoading || isLoading)}>
+                {this.context.t('my.network.invite-user')}
+            </button>;
+        const requiered = !(emailsFromCSV.length > 0 || email !== '');
```
Comment (bobeal) [inline src/main/resources/public/js/components/forms/organization-invitation-form.js]: s/requiered/required

File: src/main/resources/public/js/components/forms/organization-invitation-form.js (hunk 2 above)
Comment (zagoa) [reply src/main/resources/public/js/components/forms/organization-invitation-form.js]: That is an habit, if I want to add a dynamic class the {} are here, and in my opinion that keeps the code homogeneous.

**Pull request merged**